apps/src/p5lab/P5Lab.js

```diff
@@ -0,0 +1,1544 @@
+import $ from 'jquery';
+import _ from 'lodash';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {changeInterfaceMode, viewAnimationJson} from './gamelab/actions';
+import {startInAnimationTab} from './gamelab/stateQueries';
```

Comment:
Might be worth adding a comment explaining why we import from the gamelab directory here in case anyone stumbles on this while it's in transition.